stratum/templates/configmap-config.yaml

```diff
@@ -9,22 +9,15 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 data:
+  {{- range $key, $val := .Values.chassis_config }}
+  {{ $key }}: {{ $val | quote }}
+  {{- end }}
   stratum-config: |
     #!/bin/sh
     set -e -x
-    rm -rf $CFG_REPO
-    git clone https://$USER:$PASSWD@$CFG_SERVER/$CFG_REPO
-
-    if [ ! -z $CFG_REVIEW ] && [ ! -z $CFG_PATCHSET ]; then
-        cd $CFG_REPO
-        CFG_LAST2=$(echo $CFG_REVIEW | tail -c 3)
-        git fetch https://$USER:$PASSWD@$CFG_SERVER/$CFG_REPO refs/changes/$CFG_LAST2/$CFG_REVIEW/$CFG_PATCHSET
-        git checkout FETCH_HEAD
-        cd ..
-    fi
-
-    REMOTE_CHASSIS_CONFIG=$CFG_REPO/$CFG_FOLDER/$NODE_NAME-chassis-config.pb.txt
-    LOCAL_CHASSIS_CONFIG=chassis_config.pb.txt
+    # Mount from Helm Values, chassis_config
+    REMOTE_CHASSIS_CONFIG=/opt/configs/$NODE_NAME
```

Comment:
The word `remote` is confusing. Even for the helm value approach, it will be loaded locally onto the container via config map. I'd suggest changing this from `REMOTE` vs `LOCAL` to `FILE` vs `HELM` (or `CONFIGMAP`)